.bumpversion.cfg

```diff
@@ -9,6 +9,8 @@ tag = False
 
 [bumpversion:file:src/client/delphi_epidata.R]
 
+[bumpversion:file:src/client/delphi_epidata.py]
```

Comment:
you can remove the line for `__init__.py` a few lines below since it doesnt have the version number hardcoded in it any more